packages/maskbook/src/plugins/BarnBridge/UI/SmartYieldPoolsView.tsx

```diff
@@ -0,0 +1,176 @@
+import { makeStyles } from '@masknet/theme'
+
+import { useValueRef } from '@masknet/shared'
+import { currentAccountSettings } from '../../Wallet/settings'
+
+import { SmartYieldPoolView } from './SmartYieldPoolView'
+import type { SYPoolModelData } from './../Model/SYPoolModel'
+import type { SYPortfolioModelData } from './../Model/SYPortfolioModel'
+import { useSYPoolData, useSYPortfolioData } from '../hooks/useModels'
+
+import { useEffect } from 'react'
+import { useI18N, I18NFunction } from '../../../utils/i18n-next-ui'
+
+import PortfolioBalance from './../Import/BarnBridgeFrontEnd/PortfolioBalance'
+
+import { CardContent, CircularProgress, Paper, Tab, Tabs } from '@material-ui/core'
+import { BarnBridgeSmartYieldBarChart, BarnBridgeSmartYieldWallet } from '../BarnBridgeIcon'
+import {
+    COLOR_BARNBRIDGE_ORANGE,
+    COLOR_BARNBRIDGE_BACKGROUND_CARD_DARK,
+    COLOR_BARNBRIDGE_BACKGROUND_DARK,
+} from '../constants'
+
+import React, { useState } from 'react'
+
+const useStyles = makeStyles()((theme) => ({
+    root: {
+        display: 'flex',
+        flexDirection: 'column',
+        height: 'var(--contentHeight)',
+        padding: '0 !important',
+    },
+    tabs: {
+        backgroundColor: COLOR_BARNBRIDGE_BACKGROUND_CARD_DARK,
+        borderBottom: `solid 1px ${theme.palette.divider}`,
+        borderLeft: `solid 1px ${theme.palette.divider}`,
+        color: '#fff',
+    },
+    indicator: {
+        display: 'flex',
+        flexDirection: 'column',
+        justifySelf: 'center',
+        justifyContent: 'center',
+        alignSelf: 'centered',
+        alignItems: 'center',
+        backgroundColor: COLOR_BARNBRIDGE_ORANGE,
+    },
+    tab: {
+        display: 'flex',
+        flexDirection: 'column',
+        alignContent: 'centered',
+    },
+    icon: {
+        paddingTop: 6,
+        color: COLOR_BARNBRIDGE_ORANGE,
+    },
+    paper: {
+        backgroundColor: COLOR_BARNBRIDGE_BACKGROUND_DARK,
+        paddingTop: 10,
+    },
+    progress: {
+        color: COLOR_BARNBRIDGE_ORANGE,
+    },
+}))
+
+interface SmartYieldPoolsViewProps {}
+
+export function SmartYieldPoolsView() {
+    const [tabIndex, setTabIndex] = useState(0)
+    const [_syDataState, setPools] = useState<[SYPoolModelData, SYPortfolioModelData]>([
+        {},
+        { seniorValue: 0, juniorValue: 0 },
+    ])
+    const { t } = useI18N()
+    const walletAddress = useValueRef(currentAccountSettings)
+
+    //#region data
+    const { value: syData, error: error, loading: loadingPools, retry: retry } = useSYPoolData()
+    const {
+        value: syPortfolioData,
```

Comment:
Same here -- this is never populated. Take a look at useSYPortfolioData, it's pretty straight forward and should fetch the requested data. Please help!